.github/scripts/install-radius.sh

```diff
@@ -0,0 +1,38 @@
+# ------------------------------------------------------------
+# Copyright 2023 The Radius Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#    
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------
+
+set -xe
+
+VERSION=$1
+RAD_CLI_URL=https://raw.githubusercontent.com/radius-project/radius/main/deploy/install.sh
+RAD_CLI_EDGE_URL=ghcr.io/radius-project/rad/linux-amd64:latest
+
+if [[ $VERSION == "edge" ]]; then
```

Comment:
Do we still have edge version accessible?

Reply:
I copied this install script from samples https://github.com/radius-project/samples/blob/d611db9ee7687bf643b09570f979a0eeb99f4e25/.github/scripts/install-radius.sh#L23-L24